src/components/Dropdown/Dropdown.styles.js

```diff
@@ -150,6 +150,9 @@ const control =
       minHeight: "30px",
       border: "0 solid transparent",
       borderRadius: getCSSVar("border-radius-small"),
+      ":focus-within": {
+        boxShadow: `0 0 0 1px ${getCSSVar("primary-color")}`
```

Comment:
I see we are not defining box shadow for control for focused, is the current 'blueish' a react-select default?

Reply:
I'm asking since now the blue will probably be different than the one until now
need to make sure with Meirav that it's cool?

Reply:
<img width="1058" alt="image" src="https://github.com/mondaycom/monday-ui-react-core/assets/104433616/7c2a34a2-a946-4d6b-b107-86500624332c">

Reply:
I'm not sure you had to, was just a suggestion 😅 sorry if it sounded like a blocker by my side 😣